test/functional/eval/system_spec.lua

```diff
@@ -203,6 +203,45 @@ describe('system()', function()
       ]])
     end)
 
+    it('prints verbose information', function()
+      feed(':4verbose echo system("echo hi")<cr>')
+      screen:expect([[
+                                                             |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+        ~                                                    |
+                                                             |
+        Calling shell to execute: "echo hi"                  |
+                                                             |
+        hi                                                   |
+                                                             |
+        Press ENTER or type command to continue^              |
+      ]])
+      feed('<cr>')
+    end)
+
+    it('execution time not profiled as self time', function()
+      local tempfile = helpers.tmpname()
+
+      feed(':function! AlmostNoSelfTime()<cr>')
+      feed('echo system("sleep 2")<cr>')
```

Comment:
This just fails on Windows (no `sleep` command), so the comparison of `near(selftime, 0, 0.01)` is fragile.

I think it's enough to just check that `:profile` "did something", don't need to compare times. Though the effort is appreciated, but flakey tests are something we need to avoid (more) :)